astroquery/fermi/core.py

```diff
@@ -17,7 +17,7 @@
 @async_to_sync
 class FermiLATClass(BaseQuery):
     """
-    TODO: document
+    # TODO: document
```

Comment:
What is the reason for all the added `#` symbols?

Reply:
Okay. I will the update the issue description.

Sorry about the # symbols. I misunderstood the instruction in #281: There are many instances of "TODO" in comments in the source. Each should be given its own issue (or simply resolved) and the **TODO marked with the issue #.**